starknet_py/constants.py

```diff
@@ -7,5 +7,7 @@
     "0x041a78e741e5aF2fEc34B695679bC6891742439f7AFB8484Ecd7766661aD02BF"
 )
 
+API_VERSION = 0
```

Comment:
I wonder how will we know that the api version changed. No that there's much we can do about it.

Reply:
We will know, since it's a part of class hash calculation.
If it changes, class hashes will change.